Source/JavaScriptCore/wasm/WasmOperations.cpp

```diff
@@ -272,7 +475,7 @@ inline bool shouldJIT(unsigned functionIndex)
     return true;
 }
 
-JSC_DEFINE_NOEXCEPT_JIT_OPERATION(operationWasmTriggerTierUpNow, void, (Instance* instance, uint32_t functionIndex))
+JSC_DEFINE_NOEXCEPT_JIT_OPERATION(operationWasmTriggerTierUpNow, void, (Instance * instance, uint32_t functionIndex))
```

Comment:
Any reason to add a space between `Instance` and `*`?

Reply:
I think this was a side effect of autoformatting; I'll go ahead and revert these random changes.